neuro_san/service/http/server/http_server.py

```diff
@@ -192,9 +192,9 @@ def make_app(self, requests_limit: int, logger: EventLoopLogger):
 
         # Register templated request paths for agent API methods:
         # regexp format used here is that of Python Re standard library.
-        handlers.append((r"/api/v1/([^/]+)/function", FunctionHandler, request_initialize_data))
-        handlers.append((r"/api/v1/([^/]+)/connectivity", ConnectivityHandler, request_initialize_data))
-        handlers.append((r"/api/v1/([^/]+)/streaming_chat", StreamingChatHandler, request_initialize_data))
+        handlers.append((r"/api/v1/(.+)/function", FunctionHandler, request_initialize_data))
+        handlers.append((r"/api/v1/(.+)/connectivity", ConnectivityHandler, request_initialize_data))
+        handlers.append((r"/api/v1/(.+)/streaming_chat", StreamingChatHandler, request_initialize_data))
 
```

Comment:
Change regexp pattern so that we capture agent name in general form,
including things like abc/xyz/my_agent.

Reply:
How can we do this for gRPC?

Reply:
Interesting question. I don't exactly know right now.
I'll get to it when we have any real client using gRPC to work with neuro-san.